ios/LayoutReanimation/REASharedTransitionManager.m

```diff
@@ -489,6 +592,11 @@ - (void)setFindPrecedingViewTagForTransitionBlock:
   _findPrecedingViewTagForTransition = findPrecedingViewTagForTransition;
 }
 
+- (void)setAnimationCancellingBlock:(REAAnimationCancellingBlock)animationCancellingBlock
+{
+  _cancelLayoutAnimation = animationCancellingBlock;
+}
```

Comment:
Let's make it consistent.
```suggestion
- (void)setCancelAnimationBlock:(REAAnimationCancellingBlock)cancelAnimationBlock
{
  _cancelLayoutAnimation = cancelAnimationBlock;
}
```

Reply:
Resolved by: https://github.com/software-mansion/react-native-reanimated/pull/4043/commits/c259c6968a61439508ad110d24cfdad4d4e4c02a